_opensearch/install/plugins.md

```diff
@@ -25,7 +25,7 @@ If you are running OpenSearch in a Docker container, plugins must be installed,
 The install command takes a plugin id, which may be any of the following:
 
 - An official OpenSearch plugin name
-- Maven coordinates to a plugin zip file
+- [Maven coordinates](https://mvnrepository.com/artifact/org.opensearch.plugin) to a plugin zip file
```

Comment:
@JeffH-AWS:

i just saw these docs for the first time.

please note that this is _not_ correct (unless somebody just added that feature? but i haven't seen anything in these regards)! `opensearch-plugin install` does *not* accept maven coordinates. you cannot e.g. write `opensearch-plugin install org.opensearch.plugin:opensearch-security:2.2.0.0`! (though you could of course use the normal HTTP URL of that maven artefact, but then it's irrelevant that it's on maven)

the usage for maven here is in combination with automated build systems which need to fetch the artefacts (often through proxies in company environments), cache them and then later feed them to a build (e.g. docker build) which takes them as normal files and calls the install command with a `file:///` path

Reply:
[the blog post](https://opensearch.org/blog/technical-post/2022/08/opensearch-plugin-zips-now-in-maven-repo/) might contain further information relevant for you as it describes how the maven artefacts can be used

Reply:
Thanks for calling that out Ralph!  I was not aware that the original documentation was also inaccurate - nobody has raised an issue on that.  Would you mind opening a new issue?  You can even assign it to me, if that's something you're able to do.

Reply:
np. i've raised #1007. i don't have the rights to assign it to you, though.